src/debug/ee/debugger.h

```diff
@@ -397,7 +397,7 @@ inline LPVOID PushedRegAddr(REGDISPLAY* pRD, LPVOID pAddr)
 {
     LIMITED_METHOD_CONTRACT;
 
-#if defined(_TARGET_AMD64_)
+#ifdef WIN64EXCEPTIONS
```

Comment:
This breaks Windows ARM32 build - the _KNONVOLATILE_CONTEXT_POINTERS is not defined. It seems it should use T_KNONVOLATILE_CONTEXT_POINTERS (that's how that pCurrentContextPointers field is defined in regdisp.h)